core/src/main/resources/error/error-classes.json

```diff
@@ -757,6 +757,16 @@
       "The expression <sqlExpr> cannot be used as a grouping expression because its data type <dataType> is not an orderable data type."
     ]
   },
+  "HLL_INVALID_LG_K" : {
+    "message" : [
+      "Invalid call to <function>; the 'lgConfigK' value must be between <min> and <max>, inclusive: <value>."
+    ]
+  },
+  "HLL_UNION_DIFFERENT_LG_K" : {
+    "message" : [
+      "Sketches have different lgConfigK values: <left> and <right>. Set the 'allowDifferentLgConfigK' parameter to true to call <function> with different lgConfigK values."
```

Comment:
nit: Not sure we want to do this... Maybe put `lgConfigK` in single quotes as well?

Reply:
Sounds good, doen.